sphinx_gallery/gen_rst.py

```diff
@@ -709,7 +710,9 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf):
 
     time_m, time_s = divmod(time_elapsed, 60)
     example_nb = jupyter_notebook(script_blocks)
-    save_notebook(example_nb, replace_py_ipynb(example_file))
+    path_ipynb = replace_py_ipynb(example_file)
+    save_notebook(example_nb, path_ipynb)
```

Comment:
Why this change? Please keep in mind I also try to get pr #239 and I don't want too many things changing unnecessarily as I need to keep track of the changes.

Reply:
just cruft from an earlier iteration of this PR, reverting